hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/DeleteMetadataActor.scala

```diff
@@ -11,18 +12,16 @@ import cromwell.services.metadata.hybridcarbonite.DeleteMetadataActor.DeleteMeta
 import cromwell.services.metadata.impl.MetadataDatabaseAccess
 
 import scala.util.{Failure, Success}
-import scala.concurrent.duration._
 
-class DeleteMetadataActor(metadataDeletionConfig: MetadataDeletionConfig) extends Actor
+class DeleteMetadataActor(metadataDeletionConfig: ActiveMetadataDeletionConfig) extends Actor with StrictLogging
```

Comment:
This class already extends `ActorLogging` trait. Do we need `StrictLogging` in addition to it?

Reply:
Good point, and definitely not!